src/main/java/in/projecteka/devservice/clients/ServiceAuthenticationClient.java

```diff
@@ -63,6 +66,39 @@ private BridgeUpdateRequest bridgeRequestWith(String id, String url) {
                 .then();
     }
 
+    private BridgeServiceRequest bridgeServiceRequest(String id, String name, ServiceType type, boolean active) {
+        return new BridgeServiceRequest(id, name, type, active);
+    }
+
+    public Mono<Void> upsertBridgeServiceEntry(String bridgeId, String id,
```

Comment:
rename method name to populate/upsert BridgeServiceEntry as it is doing both add & update

Reply:
it is upsertBridgeServiceEntry